server_setup.md

```diff
@@ -0,0 +1,28 @@
+# Raspberry Pi Web Server Set Up
+
+**Prerequisites** : You must have Raspbian or Raspbian Lite installed on your Raspberry Pi, and have it connected to the internet
+
+1. **Download the setup folder to your Raspberry Pi**
+
+2. **Move into the directory where you saved the setup folder**
+     
+    For example: *`cd /home/pi/setup`*
+
+3. **Run the following command**
+    *`sudo bash ./hotspotServerSetup`*
+
+4. **The script will now start setting things up. Wait for it to do it's thing**
+
+    * If you are connecting to your Pi over ssh, your connection will drop after *`Stopping dhcpcd`*, but everything is fine. Wait about a minute or so before restarting your Pi
+
+    * If you are connected to your Pi via HDMI and a keyboard/mouse, you should see all the messages from the script, and will know when it is finished
+
+5. **Once the script has finished, reboot your Raspberry Pi by unplugging the power cord, and plugging it back in**
```

Comment:
Do we actually need to reboot it by removing the power cable? Can we add a command prompt saying that it will reboot in 5 seconds using `sudo shutdown -r 5` to reboot automatically?

Reply:
Sure, I can make the script reboot itself